zipkin/src/main/java/com/linecorp/armeria/common/tracing/RequestContextCurrentTraceContext.java

```diff
@@ -50,9 +50,11 @@
      * Use this singleton when building a {@link brave.Tracing} instance for use with
      * {@link HttpTracingService} or {@link HttpTracingClient}.
      *
+     * <p>If you need to customize the context, use {@linkplain #newBuilder()} instead.
+     *
      * @see brave.Tracing.Builder#currentTraceContext(brave.propagation.CurrentTraceContext)
      */
-    public static final CurrentTraceContext INSTANCE = new RequestContextCurrentTraceContext();
+    public static final CurrentTraceContext INSTANCE = new RequestContextCurrentTraceContext(new Builder());
```

Comment:
How about renaming to `DEFAULT`?

Reply:
INSTANCE has been released.. should we still do this?

Reply:
That's a good question. How about deprecating `INSTANCE` and adding `DEFAULT`?

Reply:
works for me